Source/WebCore/style/PropertyCascade.h

```diff
@@ -41,7 +41,7 @@ class PropertyCascade {
 public:
     enum IncludedProperties { All, InheritedOnly, AfterAnimation, AfterTransition };
 
-    PropertyCascade(const MatchResult&, CascadeLevel, IncludedProperties, const HashSet<AnimatableProperty>* = nullptr);
+    PropertyCascade(const MatchResult&, CascadeLevel, IncludedProperties, const HashSet<AnimatableProperty>* animatedProperties = nullptr);
```

Comment:
the variable name doesn't add much

Reply:
Right, that was an oversight after removing the additional parameter in a previous iteration of the patch.